test/integration/menu_administration_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class MenuAdministrationTest < ActionDispatch::IntegrationTest
+  # General thoughts
+  # Is it preferred to check the modifications directly in the database instead
+  # of querying the API for this information?
+
+  test 'add menu item' do
+    old_count = fetch_menu_items.size
+
+    # Add new menu item
+    post '/api/v1/menu', params: {item: {title: 'Test item'}}, as: :json
+
+    new_count = fetch_menu_items.size
+
+    assert_equal old_count + 1, new_count, 'Menu item was not created'
+  end
+
+  test 'update menu item' do
+    items = fetch_menu_items
+    item_to_update = items.first
+
+    # Update title for menu item
+    put api_v1_menu_url(item_to_update['id']), params: {item: {title: 'New title'}}, as: :json
```

Comment:
Consider using actual endpoint url instead of alias (see line 12). This is more explicit and since the webapp uses static endpoint declarations it is also more logical.